docs/gallery/example_term_set.yaml

```
@@ -0,0 +1,24 @@
+id: pynert/termset/species_example
+name: Species
+version: 0.0.1
+prefixes:
+  NCBI_TAXON: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=
+imports:
+  - linkml:types
+default_range: string
+
+enums:
+  Species:
+    permissible_values:
+      Homo sapiens:
+        description: description
```

```suggestion
        description: the species is human
```
Please add a description for these instead of a placeholder